src/Person/Person.jsx

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
you most likely need to disable these for the entire project. do that in .eslintrc file